modules/v1_18_R2/src/main/java/net/countercraft/movecraft/compat/v1_18_R2/IWorldHandler.java

```diff
@@ -247,6 +281,35 @@ public void setBlockFast(@NotNull Location location, @NotNull MovecraftRotation
         setBlockFast(world, BlockPos, blockData);
     }
 
+    private boolean isRedstoneComponent(Block block) {
```

Comment:
Rather than putting this in each NMS, we should put it in `net.countercraft.movecraft.util.Tags` as `REDSTONE_COMPONENTS`.  This will make it far faster than comparing the block data class type.

Reply:
no, because its looking at the NMS/Native blockstates